mythril/laser/ethereum/transaction/transaction_models.py

```diff
@@ -178,11 +149,9 @@ def initial_global_state(self) -> GlobalState:
             self.code,
             calldata_type=self.call_data_type,
```

Comment:
There should also be no calldatatype

Reply:
Keep it there (I think its actually an unused variable, so we can remove it in an other pr)

Reply:
:+1: Not going to touch it